primitives/core/src/crypto.rs

```diff
@@ -38,14 +36,15 @@ pub use secrecy::ExposeSecret;
 #[cfg(feature = "std")]
 pub use secrecy::SecretString;
 use sp_runtime_interface::pass_by::PassByInner;
-#[cfg(feature = "std")]
-use sp_std::convert::TryInto;
 #[doc(hidden)]
 pub use sp_std::ops::Deref;
 use sp_std::{convert::TryFrom, hash::Hash, str, vec::Vec};
 /// Trait to zeroize a memory buffer.
 pub use zeroize::Zeroize;
 
+#[cfg(feature = "full_crypto")]
+pub use ss58_registry::{from_known_address_format, Ss58AddressFormat, Ss58AddressFormatRegistry};
```

Comment:
How much would break when we don't re-export this here public?

If we release the ss58-registry to crates.io, it would mean that each time we do a new release because of new addresses, we need to bump the major version. However, I would like to prevent that this ripples through everything.

Reply:
The way we've defined the enum to be non-exhaustive, would adding an extra address type be a patch release as it's now a backward compatible change?

Reply:
Ahh yeah, good point. But IDK.

Reply:
removing the `pub` seems healthy to do on substrate. I'll check the impact on polkadot...

Reply:
It is in the public api anyway, because we have these functions around the default ss58 version. So, I think we can keep it. When we only bump patch versions mainly, I think it is fine.